eval_semantic_search/run_eval.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+"""
+Semantic search evaluation script - handles single config files or directories.
+"""
+
+import argparse
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+
+# Add parent directory to path
+sys.path.insert(0, str(Path(__file__).parent.parent))
```

Comment:
here here lies danger! consider either making this either:
* https://python-poetry.org/docs/pyproject/#scripts
* https://docs.djangoproject.com/en/5.2/howto/custom-management-commands/